lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -115,6 +119,8 @@ public function __construct(Configuration $configuration, $version, $class)
         $this->platform = $this->connection->getDatabasePlatform();
         $this->migration = new $class($this);
         $this->version = $version;
+        $schemaManipulator = new SchemaDiffProvider($this->sm, $this->platform);
```

Comment:
@mikeSimonson removing private properties is not a BC break: that's the beauty of making things `private` ;-)